projects/common/vpk180/vpk180_system_constr.xdc

```diff
@@ -0,0 +1,29 @@
+###############################################################################
+## Copyright (C) 2021-2023 Analog Devices, Inc. All rights reserved.
+### SPDX short identifier: ADIBSD
+###############################################################################
+
+create_clock -period 5.000 -name sys_clk_p [get_ports sys_clk_p]
+
+
+set_property	PACKAGE_PIN	BK5 	  [get_ports sys_clk_n]
+set_property	PACKAGE_PIN	BK6	    [get_ports sys_clk_p]
+
+# Define SPI clock
+create_clock -name spi0_clk      -period 40  [get_pins -hier */EMIOSPI0SCLKO]
+create_clock -name spi1_clk      -period 40  [get_pins -hier */EMIOSPI1SCLKO]
+
+# GPIOs
+# (switches, leds and such)
+set_property -dict {PACKAGE_PIN BA49 IOSTANDARD LVCMOS15} [get_ports gpio_led[0]] ;
```

Comment:
This is just a suggestion, but would it be better if there would be no space between the `]` and `;`?